pymathics/natlang/main.py

```diff
@@ -484,12 +451,12 @@ def eval(self, text: String, word, evaluation: Evaluation, options: dict):
             words = [w.lower() for w in words]
         n = 0
         for token in doc:
-            text = token.text
+            token_text = token.text
             if ignore_case:
-                text = text.lower()
-            if text in words:
+                token_text = token_text.lower()
+            if token_text in words:
                 n += 1
-        return Expression(SymbolN, Integer(n) / Integer(len(doc)))
+        return eval_N(Integer(n) / Integer(len(doc)), evaluation)
```

Comment:
Here we could use `eval_N`

Reply:
I don't understand - that is what is currently used.

Reply:
Sorry, I didn't see the change.